include/aspect/boundary_traction/ascii_data.h

```diff
@@ -95,6 +115,19 @@ namespace aspect
 
       private:
         std::set<types::boundary_id> boundary_ids;
+
+        /**
+         * Whether to specify traction in x, y, z components, or
+         * r, phi, theta components.
+         */
+        bool use_spherical_unit_vectors;
+
+        /**
+        * Whether to prescribe pressure (default: true) or full traction vector (false)
+        * at the boundary. If true, only 1 component will be used for the boundary condition.
+        */
+        bool prescribe_pressure_instead_of_full_traction=true;
```

Comment:
This is not wrong, but it is not our usual code style to put the initialization into the class header. Could you move the initialization into the initializer of the constructor in the source file? See e.g. how `boundary_traction/function.cc` does this. This is purely a change to maintain the visual consistency and layout of the code, there is nothing wrong with the code you wrote.